code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -125,18 +125,16 @@
 	return (exposed_temperature >= heat_capacity || to_be_destroyed)
 
 /turf/atmos_expose(datum/gas_mixture/air, exposed_temperature)
-	if(exposed_temperature >= heat_capacity)
+	if(exposed_temperature >= max_fire_temperature_sustained)
 		to_be_destroyed = TRUE
-	if(to_be_destroyed && exposed_temperature >= max_fire_temperature_sustained)
-		max_fire_temperature_sustained = min(exposed_temperature, max_fire_temperature_sustained + heat_capacity / 4) //Ramp up to 100% yeah?
```

Comment:
undocc'd change, a hot fire will now instantly run full melting chance instead of slowly heating up (simulating the turf warming)